tooltool.py

```diff
@@ -937,6 +947,11 @@ def main(argv, _skip_logging=False):
                            'files that cannot be distributed out of the company '
                            'but not for secrets; "public" files are available to '
                            'anyone withou trestriction')
+    parser.add_option('--ttl', default=None,
+                      type='int', dest='ttl', action='store',
+                      help='Time to live for the file to uploaded, in days; the '
+                           'files will be removed from storage, but the metadata '
+                           'will be retained')
```

Comment:
Also, mention how to specify a "forever" TTL in this option, and that "forever" is always considered longer than a finite time.